pulp_python/app/models.py

```diff
@@ -170,9 +179,12 @@ def _parse_metadata(cls, project, version, distribution):
         """
         Create a dictionary of metadata needed to create a PythonContentUnit from
         the project, version, and distribution metadata
-        :param project:
-        :param version:
-        :param distribution:
+
+        Args:
```

Comment:
Optional tweaks:

project (dict): of metadata relevant to the entire Python project
version (string): version of distribution
distribution (dict): of metadata of a single Python distribution